README.md

```diff
@@ -24,13 +24,49 @@ Feel free for:
 
 - Create Issue, Request Features, & send a pull request.
 
-### Make new packages
+## How to use
+
+- clone this repo
+
+```
+  git clone https://github.com/kodingdotninja/chakra-icons.git
+```
+
+- go to the project directory
+
+```
+  cd chakra-icons
+```
+
+- install dependencies
+
+```
+  yarn | yarn install | npm install
```

Comment:
cause we use `yarn`. don't need `npm install`

Reply:
```suggestion
  yarn | yarn install
```

Reply:
oke I got it